Week04/MoviesApp/app/src/main/java/tech/danielwaiguru/moviesapp/MovieFragment.kt

```diff
@@ -0,0 +1,41 @@
+package tech.danielwaiguru.moviesapp
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import kotlinx.android.synthetic.main.fragment_movie.view.*
+
+
+class MovieFragment : Fragment() {
+    private lateinit var movieRecyclerView: RecyclerView
+    private lateinit var movieAdapter: MovieAdapter
+
+    override fun onCreate(savedInstanceState: Bundle?) {
```

Comment:
Pay attention to the warnings from the IDE. There is no reason to include an override if you are not doing anything other than calling super.